### ISSUE TITLE
Feature request: use deep autoencoders for dimensionality reduction

### ISSUE THREAD

**Issue by andrewheusser** [ContextLab/hypertools#118]
using deep autoencoders to do dimensionality reduction could be a unique way to visualize high-dim data:

-info about deep autoencoders: https://deeplearning4j.org/deepautoencoder
-python implementation using tensorflow: https://github.com/cmgreen210/TensorFlowDeepAutoencoder
-geoff hinton lecture with deep autoencoder data viz: https://www.youtube.com/watch?v=ARQ6PZh8vgE&list=PLoRl3Ht4JOcdU872GhiYWf6jwrk_SNhz9&index=69#t=386.907936

**Comment by andrewheusser** [comment]
<img width="543" alt="screen shot 2017-05-23 at 10 34 42 pm" src="https://cloud.githubusercontent.com/assets/6596567/26384660/d4cbd594-4008-11e7-8281-8a4ee06c9fa4.png">

**Comment by andrewheusser** [comment]
input and output are trained to look as similar as possible

**Comment by andrewheusser** [comment]
https://blog.keras.io/building-autoencoders-in-keras.html

**Comment by jeremymanning** [comment]
Ah...this issue redundant with [this issue](https://github.com/ContextLab/hypertools/issues/162).  I've spelled things out a bit more in the newer version, so I'm going to close this one.
